src/IECoreRI/RendererImplementation.cpp

```diff
@@ -1823,12 +1824,69 @@ void IECoreRI::RendererImplementation::emitCurvesPrimitive( const IECore::Curves
 				(char *)( primitive->periodic() ? "periodic" : "nonperiodic" ),
 				pv.n(), pv.tokens(), pv.values() );
 }
-	
+
+namespace
+{
+
+// note this is duplicated from src/IECore/Primitive.cpp
```

Comment:
Once we've moved to indexed V2fs for UVs, I think the duplication will be gone actually. The Primitive loading code will be doing the conversion to V2f and indices at the same time as unflipping, and this function here will just be dealing with the V2f form. I still think `MeshAlgo::flipT( MeshPrimitive *primitive )` would be a reasonable thing to have in a public header though, should we need to share it somewhere else.

Reply:
Lets leave this for the next (or next next) PR, since as you say, the function will be changing drastically.